src/app/pages/AllePage/AllePage.stories.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import AllePage from "./AllePage";
```

Comment:
Source-code, components and files should be in English.
`AllPage` or `LandingPage`?